yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java

```diff
@@ -24,18 +24,18 @@
     /**
      * Equal to ${given_names} + " " + ${family_name}.
      *
-     * @return the given names + the surname
-     * @deprecated this method has never featured as intended (it has always returned null). Deprecated since version 1.2, likely to be removed in version 2.0. Instead please use {@link #getGivenAndLastNames()}
+     * @return the full name
      */
-    @Deprecated
     String getFullName();
 
     /**
      * Equal to ${given_names} + " " + ${family_name}.
      *
      * @return the given names + the surname
      * @since 1.2
+     * @deprecated please use {@link #getFullName() instead}
```

Comment:
I don't think we should suggest to use getFullName instead of this.
This one is a convenience method that returns the concatenation of given_names and family_name.
`getFullName` returns the full_name attribute.

If the qrcode requests given_names and family_name, full_name will not be there.
Similarly, if the qr code requests full_name, then given_names and family_name will not be there.

The reason for having these 3 attributes (and specifically full_name ) is that certain ID documents like the Indian Aadhaar do not provide the name components, but only the full_name.

Reply:
Ah, I thought that would `GetFullName` would return the equivalent of `GetGivenAndLastName`, if only the `Given Name` and `Family Name` are present (and not `FullName`). 

If that's not the case, do we still want to deprecate `GetGivenAndLastName`, if it's providing us with a helper method which we're not getting otherwise?

Reply:
We don't really need to deprecate it although I don't how much value it actually adds.
@bucky-boy  thoughts?

Reply:
I think we need to deprecate it, it was my suggestion.
We now have getFullName, which currently describes itself as "${given_names} + " " + ${family_name}" as well as getGivenAndLastNames.  I'd suggest we try to keep it trimmed down

Reply:
I agree, I think we can deprecate getGivenAndLastNames, making it simple, and if the client wants to add a similar method they can do this easily.